app/views/course_sessions/show.html.erb

```diff
@@ -1,73 +1,34 @@
-<p id="notice"><%= notice %></p>
-<div class="" style="border:1px solid black;">
-  <p>
-    <strong>Session Name:</strong>
-    <%= @course_session.name %>
-  </p>
-
-  <p>
-    <strong>Description:</strong>
-    <%= @course_session.description %>
-  </p>
-
-  <p>
-    <strong>Start Date:</strong>
-    <%= @course_session.start_date %>
-  </p>
-
-  <p>
-    <strong>End Date:</strong>
-    <%= @course_session.end_date %>
-  </p>
-
-  <p>
-    <strong>Github Repository:</strong>
-    <%= @course_session.repository_link %>
-  </p>
-
-  <%= link_to 'Edit', edit_course_session_path(@course_session) %> |
-  <%= link_to 'Back', course_sessions_path %>
+<div class="container mb-auto py-3 text-center">
+    <h1><%= @course_session.name %></h1>
+    <h3><%= @course_session.description %></h3>
+    <%= link_to fa_icon("github"), @course_session.repository_link %> |
+    <%= link_to 'Edit', edit_course_session_path(@course_session) %> |
+    <%= link_to 'Back', course_sessions_path %>
 </div>
-
-<br></br>
-
-<p>
-  <strong>Lectures</strong>
-</p>
-<table>
-  <thead>
-    <tr>
-      <th>Lecture ID</th>
-      <th>Lecture name</th>
-      <th>Lecture content</th>
-      <th>Video link</th>
-      <th>Homework</th>
-      <th>Teacher</th>
-      <th>CarrotU Session</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
+<p id="notice"><%= notice %></p>
+  <div class="container mx-auto">
```

Comment:
nit: I think this was supposed to be at the same indentation as the notice block

Reply:
Fixed indentation.